### DISCUSSION TITLE
Regaining control of lost GitHub profile: is it possible?

### DISCUSSION THREAD

**Discussion by scotty3005** [community/community#73310] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 

I have a GitHub profile that I have created 3 years ago while I was working for a certain company. That user did not really do any work on GitHub  it was just a required for some working tasks. Now I would like to reuse that username. Considering that the account is still existing ans it is linked to my deleted work email of that time, do I have any possibility to request the deletion of that account in order to create a new one with the same username?

Thanks in advance for your help

**Comment by haxybaxy** [comment]
If you want to reuse the username associated with your old GitHub account and the account is still active, you generally cannot request to delete it to create a new one with the same username. GitHub has specific policies and procedures regarding account deletion, and they typically do not delete accounts unless they violate GitHub's terms of service.

If you no longer have access to the old email address and cannot log in to the account, you can try contacting GitHub Support for assistance. They may be able to help you recover or regain access to the old account. Be prepared to provide as much information as possible to prove that you are the legitimate owner of the account.

It's important to note that creating multiple GitHub accounts is generally discouraged, and GitHub may take action if they detect violations of their terms of service or policies. It's best to work with the existing account if possible or follow GitHub's guidelines for resolving account access issues.
